.github/CONTRIBUTING.md

```diff
@@ -12,17 +12,59 @@ These are mostly guidelines, not rules. Use your best judgment, and feel free to
 - Make sure that after the changes you don't get any [markdownlint](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md) errors
 - If the `metadata.json` and `metadata.schema.json` needs changes, announce it first in the Discord `#development` channel and ping`voidpointer` and `nitsua` so the 3rd party apps don't break :bangbang:
 
+## Branch naming
+
+We're trying to be as consistent as possible for automation of the changelog.
+
+- **feat/xxx** Commits, that adds a new feature
+- **fix/xxx**  Commits, that fixes a bug/issue
+- **style/xxx** Commits, that do not affect the meaning (white-space, formatting, missing semi-colons, etc)
+- **chore/xxx** Miscellaneous commits (administrative / cleanup / backend)
+
+Where 'xxx' can stand for Starr/Sonarr/Radarr/Guide etc
+
+*Examples:*
+`feat/Starr-ELEANOR-to-Scene-CF`
+`feat/Radarr-CiNEPHiLES-to-Remux-Tier-02`
+`feat/maintenance`
+`fix/dv-hdr10-cf-should-match-dv-hdr10+`
+
+[Source](https://gist.github.com/qoomon/5dfcdf8eec66a051ecd85625518cfd13#types){:target="_blank" rel="noopener noreferrer"}
+
+## Pull Request naming
+
+We're trying to be as consistent as possible for automation of the changelog.
+
+- **feat(xxx):** Commits, that adds a new feature
+- **fix(xxx):**  Commits, that fixes a bug/issue
+- **style(xxx):** Commits, that do not affect the meaning (white-space, formatting, missing semi-colons, etc)
+- **chore(xxx):** Miscellaneous commits (administrative / cleanup / backend)
+
+Where 'xxx' can stand for Starr/Sonarr/Radarr/Guide etc
+
+*Examples:*
+`feat(starr): add ELEANOR to Scene CF`
+`feat(radarr): add CiNEPHiLES to Remux Tier 02`
+`feat(maintenance): various changes to the Starr CFs`
+`fix(starr): DV HDR10 CF should match DV HDR10+`
+
+[Source](https://gist.github.com/qoomon/5dfcdf8eec66a051ecd85625518cfd13#types){:target="_blank" rel="noopener noreferrer"}
+
+## When doing a PR that is WIP
+
+When doing a PR that is in-progress and not yet complete / ready for review not yet done fully please ensure it is a **DRAFT Pull Request**
```

Comment:
`When doing a PR that is in-progress and not yet complete / ready for review not yet done fully please ensure it is a **DRAFT Pull Request**` to `When doing a PR that is in-progress and not yet complete / ready for review or not yet done fully, please ensure it is a **DRAFT Pull Request**`
I think this would make it more understandable